2420/2/versus.rb

```diff
@@ -0,0 +1,69 @@
+# :reek:FeatureEnvy
+
+# class Battler
+class Battler
```

Comment:
Do you read about SRP? 
http://jjbohn.info/blog/2014/07/28/single-responsibility-principle-a-solid-week/